src/profile-logic/marker-schema.js

```diff
@@ -45,6 +45,20 @@ export const markerSchemaFrontEndOnly: MarkerSchema[] = [
       },
     ],
   },
+  {
+    name: 'IPC',
```

Comment:
Since we also have IPC marker schema coming from the back-end, it would be good to note that we are actually overwriting this marker schema here.